knative-operator/deploy/resources/kn-downloads.yaml

```diff
@@ -0,0 +1,155 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: serverless-operator
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: serverless-operator
+  name: kn-downloads
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: serverless-operator
+      component: kn-downloads
+  template:
+    metadata:
+      name: kn-downloads
+      labels:
+        app: serverless-operator
+        component: kn-downloads
+    spec:
+      nodeSelector:
+        kubernetes.io/os: "linux"
```

Comment:
Why is this needed?

Reply:
again, coming as is from oc spec, will remove it.